misc/dbt-materialize/tests/adapter/test_data_tests.py

```diff
@@ -47,3 +47,45 @@ def test_store_failures(self, project):
 
         result_statuses = sorted(r.status for r in results)
         assert result_statuses == ["fail", "pass"]
+
+
+class TestDataTestCluster:
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "data_tests",
+            "tests": {
+                "test": {
+                    "+store_failures": True,
+                    "+schema": "etl_failure",
+                    "+cluster": "not_default_test",
+                },
+            },
+        }
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "test_materialized_view.sql": test_materialized_view,
+        }
+
+    @pytest.fixture(scope="class")
+    def tests(self):
+        return {
+            "unique.sql": unique,
+        }
+
+    def test_store_failures_cluster(self, project):
+        project.run_sql("CREATE CLUSTER not_default_test REPLICAS (r1 (SIZE '1'))")
```

Comment:
This syntax is still valid, but we should start using the managed cluster syntax, and eventually refactor all tests that use the legacy one.

`CREATE CLUSTER not_default_test SIZE = '1';`

Reply:
Made a separate ticket and will update those all today https://github.com/MaterializeInc/materialize/issues/21935